lib/services/debug-service.ts

```diff
@@ -35,8 +35,10 @@ export class DebugService extends EventEmitter implements IDebugService {
 			projectDir: debugData.projectDir
 		});
 
-		if (!(await device.applicationManager.isApplicationInstalled(debugData.applicationIdentifier))) {
-			this.$errors.failWithoutHelp(`The application ${debugData.applicationIdentifier} is not installed on device with identifier ${debugData.deviceIdentifier}.`);
+		const devicePlatform = device.deviceInfo.platform;
+
+		if (!(await device.applicationManager.isApplicationInstalled(debugData.applicationIdentifier[devicePlatform.toLowerCase()]))) {
```

Comment:
please change all the `toLowerCase()` all over the PR to a commonplace.

Reply:
Well I discussed the options with @PanayotCankov and @rosen-vladimirov and this was the least bad way to handle it.

The other was to make a getter and setter for the lowercase properties inside an implementation of IProjectIdentifier.

Reply:
Given the platforms and other constants are living only in our CLI code, I don't understand the issue of not constantly using `toLowerCase`?